modules/understanding-vsphere-problem-detector.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_vsphere/using-vsphere-problem-detector-operator.adoc
+
+:operator-name: vSphere Problem Detector Operator
+
+[id="understanding-vsphere-problem-detector_{context}"]
+= Understanding the vSphere Problem Detector Operator
+
+The {operator-name} checks clusters that are deployed on vSphere for common installation and misconfiguration issues that are related to storage.
+
+The Operator runs in the `openshift-cluster-storage-operator` namespace and is started by the `cluster-storage-operator` when the Cluster Storage Operator detects that the cluster is deployed on vSphere. The {operator-name} communicates with the vSphere vCenter Server to determine the virtual machines in the cluster, the default datastore, and so on. The Operator uses the credentials from the `cloud-credentials-provider` to connect to vSphere.
+
+The Operator runs the checks periodically. The schedule is as follows:
+
+* The checks run every 8 hours.
+
+* If any check fails, the Operator runs the checks again in intervals of 1 minute, 2 minutes, 4, 8, and so on. The Operator doubles the interval up to a maximum interval of 8 hours.
+
+* When all checks pass, the schedule returns to an 8 hour interval.
+
+The Operator increases the frequency of the checks after a failure so that the Operator can report success quickly after the failure condition is remedied. You can run the Operator manually for immediate troubleshooting information.
+
+// Clear temporary attributes
+:operator-name:
```

Comment:
```suggestion
:!operator-name:
```
You need an exclamation point in the statement to unset an attribute. See http://asciidoc.github.io/asciidoc/chunked/ch28.html

Reply:
Many thanks.  I hallucinated some doc that suggested setting it to empty was equivalent.